Apps/Src/SendTritium.c

```diff
@@ -535,7 +552,7 @@ static void OnePedalDriveHandler(){
     if(accelPedalPercent <= ONEPEDAL_BRAKE_THRESHOLD){
         // Regen brake: Map 0 -> brake to 100 -> 0
         velocitySetpoint = 0;
-        currentSetpoint = percentToFloat(map(accelPedalPercent, 0, ONEPEDAL_BRAKE_THRESHOLD, 100, 0));
+        currentSetpoint = percentToFloat(map(accelPedalPercent, PEDAL_MIN, ONEPEDAL_BRAKE_THRESHOLD, CURRENT_SP_MAX, CURRENT_SP_MIN));
```

Comment:
Does using pedal_min satisfy any noise tolerance concerns? IIRC, we have the ACCEL_PEDAL_THRESHOLD at 10% to ensure that ppl are pressing it for sure. May want to adjust the boundaries somewhat. May also need testing to confirm this.